<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnStart">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>250</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>开始游戏</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="btnLoad">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>310</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>载入游戏</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="btnRestart">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>370</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>重新开始</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="btnPause">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>430</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>暂停游戏</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="btnCon">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>490</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>继续游戏</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="btnSave">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>550</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>保存游戏</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="btnQuit">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>610</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>退出游戏</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>140</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>70</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>时间</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>选项</string>
    </property>
    <addaction name="separator"/>
    <addaction name="start"/>
    <addaction name="load"/>
    <addaction name="restart"/>
    <addaction name="separator"/>
    <addaction name="pause"/>
    <addaction name="con"/>
    <addaction name="separator"/>
    <addaction name="save"/>
    <addaction name="quit"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="start"/>
   <addaction name="load"/>
   <addaction name="restart"/>
   <addaction name="pause"/>
   <addaction name="con"/>
   <addaction name="save"/>
   <addaction name="quit"/>
  </widget>
  <action name="start">
   <property name="icon">
    <iconset resource="myresource.qrc">
     <normaloff>:/icon/satrt.png</normaloff>
     <normalon>start.jpg</normalon>:/icon/satrt.png</iconset>
   </property>
   <property name="text">
    <string>开始游戏</string>
   </property>
  </action>
  <action name="load">
   <property name="icon">
    <iconset resource="myresource.qrc">
     <normaloff>:/icon/load.png</normaloff>:/icon/load.png</iconset>
   </property>
   <property name="text">
    <string>载入游戏</string>
   </property>
  </action>
  <action name="restart">
   <property name="icon">
    <iconset resource="myresource.qrc">
     <normaloff>:/icon/restart.png</normaloff>
     <normalon>:/icon/restart.png</normalon>:/icon/restart.png</iconset>
   </property>
   <property name="text">
    <string>重新开始</string>
   </property>
  </action>
  <action name="pause">
   <property name="icon">
    <iconset resource="myresource.qrc">
     <normaloff>:/icon/pause.png</normaloff>:/icon/pause.png</iconset>
   </property>
   <property name="text">
    <string>暂停游戏</string>
   </property>
  </action>
  <action name="con">
   <property name="icon">
    <iconset resource="myresource.qrc">
     <normaloff>:/icon/continue.png</normaloff>:/icon/continue.png</iconset>
   </property>
   <property name="text">
    <string>继续游戏</string>
   </property>
  </action>
  <action name="save">
   <property name="icon">
    <iconset resource="myresource.qrc">
     <normaloff>:/icon/save.png</normaloff>:/icon/save.png</iconset>
   </property>
   <property name="text">
    <string>保存游戏</string>
   </property>
  </action>
  <action name="quit">
   <property name="icon">
    <iconset resource="myresource.qrc">
     <normaloff>:/icon/quit.png</normaloff>:/icon/quit.png</iconset>
   </property>
   <property name="text">
    <string>退出游戏</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="myresource.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
